clients/banking/src/utils/exchanges/suspenseDedupExchange.ts

```diff
@@ -0,0 +1,33 @@
+import { Exchange, Operation, OperationResult, makeOperation } from "@urql/core";
+import { map, pipe, tap } from "wonka";
+
+const RENDER_LEEWAY = 500;
+
+export const suspenseDedupExchange: Exchange = ({ forward }) => {
```

Comment:
I think a little comment block explaining the issue with Suspense, why there are 2 requests sended and how it might be solved with the future suspense cache in react